e2e/testcases/cli_test.go

```diff
@@ -1331,6 +1331,25 @@ func TestNomosMigrate(t *testing.T) {
 		},
 	}
 
+	nt.T.Cleanup(func() {
```

Comment:
Besides deleting ConfigManagement object in a cleanup block, do we also need to delete it before creating?
I'm thinking of a case when the cleanup block failed to be executed.

Reply:
In that case the `Create` call will fail but the `Cleanup` block will delete the object and the test will pass on the next execution. We could add a duplicate delete call but I think this is sufficient

Reply:
ACK. The `Clean` function will also delete test data before all tests start.